.golangci.yaml

```diff
@@ -112,7 +112,12 @@ linters:
           - gosec
           - lll
           - mnd
+          - funlen # it's ok for the examples to have big main() functions
         path: examples/
+      - linters:
+          - gocritic #commentedOutCode: "curve.SignFelts..." is there just as an explanation
+          - unused # it's part of the example to have two unused functions
+        path: examples/paymaster/
```

Comment:
I don't think adding this exception is good. Why don't you add the lint ignore directly to the file?